be/src/olap/reader.h

```diff
@@ -128,7 +128,7 @@ class Reader {
     }
 
     uint64_t filtered_rows() const {
-        return _stats.rows_del_filtered;
```

Comment:
Hi, is it compatible with segment v1?
In alpha_rowset_reader.h, filtered_rows() is only calculate rows_del_filtered.

Reply:
Yes, it is compatible. Because rows_conditions_filtered is only used for SegmentV2, so it should always to 0 in Segment V1.